index.html

```diff
@@ -4629,14 +4651,15 @@ <h3>Bearer Credentials</h3>
         </p>
       </section>
 
-      <section class="informative">
-        <h3>Validity Checks</h3>
+      <section class="normative">
+        <h3>Validation</h3>
```

Comment:
There is already a section called `Validation`. Right now it's appendix A. Changing this line will update all the existing anchor tags of the form `<a href="#validation">`, and they'll start pointing to this section, instead of the appendix. I've noted what you'd need to do to update one such example below.

Reply:
I don't think its wise to make validation normative... it drastically increases test complexity, and it will trample on use cases.

Reply:
@andresuribe87 I switched back to informative, as requested in yesterday's call. 

However, I'm not sure how to handle your suggestion. It seemed like you made it normative, which added the link problem. Does this problem still exist if this is non-normative?

Reply:
Except that verifiers are free to use their own rules and disregard validFrom and validUntil in cases where expiry doesn't matter.

The requirement isn't that verifiers must accept a VC as valid when the current timestamp is between validFrom and validUntil. The requirement is that verifiers must understand whether or not the validFrom or validUntil properties matter to their use case, and how.  That requirement is true for all properties a verifier relies on.

Once they understand their own desired policy, then they can build their validation rules to enforce that policy. How they come up with that policy and how it is enforced is not the W3C's business. It is literally the verifier's business.

Reply:
you can resolve any thread with me, you feel is stale.